spyder/utils/syntaxhighlighters.py

```diff
@@ -356,7 +356,10 @@ def get_token(self):
 class PythonSH(BaseSH):
     """Python Syntax Highlighter"""
     # Syntax highlighting rules:
-    PROG = re.compile(make_python_patterns(), re.S)
+    add_kw = []
+    if PY3:

```

Comment:
Let's add `async` and `await` regardless of the Python version. This is because Spyder can be running under Python 2 but used to display Python 3 files.